website/case-studies/implementation-of-flint-modules.md

```diff
@@ -0,0 +1,407 @@
+5.  [WOFOST crop growth module](#wofost-crop-growth-module)
+6.  [Turnover module](#turnover-module)
+7.  [Decomposition module](#decomposition-module)
+8.  [RothC soil carbon module](#rothc-soil-carbon-module)
+9.  [Chapman Richards module](#chapman-richards-module)
+10. [GCBM module](#gcbm-module)
+11. [Frequently asked questions](#frequently-asked-questions)
+
+## Prerequisites
+
+These are the prerequistes you need to know before learning about **Modules**. This section presumes that you are already familiar with **FLINT**. If not, see [the FLINT repository](https://github.com/moja-global/FLINT) to better understand **FLINT** before continuing.
+
+## How do modules work?
+
+Each module reads (or is provided with – see Unit Controller) information about the current state variables and the data required to update the state variables, such as climate data or information about disturbances to simulate. Each module then performs the required calculations and returns information about the updates to apply to each of the state variables and C pools. 
+
+For state variables such as age or height it is possible for modules to return the updated value but for all carbon pools and other fluxes the module returns an array (sparse matrix) of the proposed operations. This array will include the information about the source pool, the sink pool, and the amount (Mg ha-1 per time step) of the flux. Module-specific metadata regarding units and time step size are also required. This information is all returned to the Unit Controller.
+
+In terms of the FLINT programming framework, there are internal and external modules. The internal modules tend to be generic utility modules, for example, for producing output tables. While external modules are regionally or nationally specific. By using modules and separating the generic and specific, FLINT’s framework remains highly flexible and new modules can be ‘plugged in’ as they are developed.
+
+### Pools
+
+A pool is a reservoir within which something can be stored and released. For example, a carbon pool is a reservoir from which carbon can be stored (sequestered and/or maintained) or emitted, such as debris. Within FLINT, each pool has attributed a value such as tonnes carbon, and at each time step, the FLINT can move stores from one pool to another using an operation. Here are the different pools present in FLINT.
+
+|![Diagram depicting how modules determine pools in FLINT.](../static/img/pools-module-case-study.png) |
+|:--:|
+| Diagram depicting how modules determine pools in FLINT. |
+
+The **Intergovernmental Panel on Climate Change** ([IPCC](https://www.ipcc.ch/about/)) is the United Nations body for assessing the science related to climate change. FLINT has been designed to follow the IPCC Good Practice Guidance and the evolving transparency framework under the Paris Agreement. The approach is solidly rooted in science and applies a carbon balance method to ensure the generated results are reliable.
```

Comment:
This doesn't link to the describption of Pools. Do you know what it was trying to describe?

Reply:
Nope. I will just remove it from here.